cypress/e2e/pages/storyPage/tests.js

```diff
@@ -26,45 +26,52 @@ export const testsThatFollowSmokeTestConfig = ({
   isAmp,
   variant,
 }) => {
+  /** 
```

Comment:
@MeriemMechri before we go too far down the road with this, I believe that these scenarios are a perfect example of a test which can be moved to integration tests. 

We should not be hardcoding the intro data in the tests, and because we're only checking that a description / paragraph text is rendered, rather than a user interaction / user journey, then it shouldn't really be an E2E test.

I suggest that we move these tests to `src/integration/pages/storyPage` - ensuring that we use an example of each type of test:
- one where there is a block with role = 'introduction'
- another which has a block type = paragraph (note this might already exist)

Happy to work with you on this